images/router/f5/Dockerfile

```diff
@@ -1,11 +1,8 @@
 #
-# This is the HAProxy router for OpenShift Origin.
+# This is the F5 router for OpenShift Origin.
 #
 # The standard name for this image is openshift/origin-f5-router
 #
-FROM openshift/origin-base
+FROM openshift/origin
 
-ADD bin/openshift /usr/bin/openshift
-RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
-
-ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
+ENTRYPOINT ["/usr/bin/openshift-f5-router"]
```

Comment:
:+1:  for a cleaner dockerfile - but see above comment on moving the symlink for ```openshift-f5-router```.